styles.css

```diff
@@ -1,155 +1,103 @@
-  flex-grow: 0;
-  top: 25%;
-}
-.let {
-  width: 100vw;
-  height: 10vw;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 500;
-  font-size: 3vw;
-  line-height: 2vw;
-  color: #7F8CFF;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
-.icon{
-display: flex;
-flex-grow: 1;
-max-width: 100vw;
-}
-.icon-1{
-  display: flex;
-  flex-grow: 1;
-  max-width: 100vw;
-  }
-.logo {
-  max-width: 40vw;
-  font-family: 'Poppins';
-  font-style: normal;
-  font-weight: 700;
-  font-size: 5vw;
-  line-height: 5vw;
-  display: flex;
-  align-items: start;
-  text-align: start;
-  color: #6070FF;
-  flex: none;
-  order: 0;
-}
-.click{
-width: 100vw;
-height: 20vw;
-flex-flow: row nowrap;
-justify-content: space-around;
-margin-left: 1vw;
-margin-top:  2vw;
-flex-grow: 1;
-}
-.click-1{
-  margin-left: 1vw;
-  margin-top:  10vw;
-}
-.top{
-display: flex;
-flex-grow: 1;
-max-width: 100vw;
-max-height: 10vw;
-}
-.top-1{
-  display: flex;
-  flex-grow: 1;
-  max-width: 100vw;
-  max-height: 30vw;
-  }
-.inner{
-margin-left: 1vw;
-margin-top:  1vw;
-}
-.bottom{
-  max-width: 10vw;
-  margin-left: 50vw;
-  margin-top: 0vw;
-}
-.bottom-3{
-  margin-left: 15vw;
-}
-.bottom-2{
-  margin-left: 30vw;
-}
-.top-el{
-  margin-left: 30vw;
-}
-.cricle{
-  margin-left: 50vw;
-}
-.top-elip{
-  margin-left: 40vw;
-}
-.twitter{
-  max-width: 3vw;
-  height: auto;
-  margin-left: 0vw;
-}
-.in{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
-.medium{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
-.github{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
-.angellist{
-  max-width: 4vw;
-  height: auto;
-  margin-left: 4vw;
-}
+main{
+  width: 100vw;
+  min-height: 100vh;
+  background-image: url("background.png");
+  background-size: 105vw 23vw;
+  background-repeat: no-repeat;
+}
+.navgation {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  max-width: 100vw;
+  max-height: 15vw;
+  align-items: center;
+}
+.menu {
+  max-width: vw;
+  max-height: 10vw;
+  margin-left: 75vw;
+}
+.para {
+  max-width: 100vw;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 5.5vw;
+  line-height: 6vw;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+.feyisa {
+  max-width: 100vw;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14vw; 
+  line-height: 13vw;
+  display: flex;
+  align-items: center;
+  color: #172B4D;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  top: 25%;
+}
+.let {
+  width: 100vw;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 3vw;
+  line-height: 2vw;
+  color: #7F8CFF;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+.part{
+  width: 100vm;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  
+  }
+.icon{
+display: flex;
+flex-grow: 1;
+max-width: 35vw;
+max-height: 20vm;
+margin-top: 5vm;
+justify-content: space-between;
+}

```

Comment:
- [ ] Ouch! this is a close one but as mentioned previously the spacing between the social media icons is still not in sync with the Figma design I'd recommend you add a left and right margin to the icons.
- Your design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/f737c15f-0066-49d8-aeff-4d410b217d8a)
- Figma design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/c04aa59d-4d84-41c2-9124-b4c2a3201dd8)